examples/quickstart-xgboost/client.py

```diff
@@ -0,0 +1,150 @@
+import warnings
+from logging import INFO
+import xgboost as xgb
+
+import flwr as fl
+from flwr_datasets import FederatedDataset
+from flwr.common.logger import log
+from flwr.common import (
+    Code,
+    EvaluateIns,
+    EvaluateRes,
+    FitIns,
+    FitRes,
+    GetParametersIns,
+    GetParametersRes,
+    Parameters,
+    Status,
+)
+
+from dataset import instantiate_partitioner, train_test_split, transform_dataset_to_dmatrix
+
+
+warnings.filterwarnings("ignore", category=UserWarning)
+
+# Load (HIGGS) dataset and conduct partitioning
+num_partitions = 20
+# partitioner type is chosen from ["uniform", "linear", "square", "exponential"]
+partitioner_type = "uniform"
```

Comment:
Let's add the `num_partitions` and the `partitioner_type` as the arguments to argpase too.

Reply:
Please see above. I think we can even do a config parser for more hyper-parameter selection